### DISCUSSION TITLE
Basic math calculations out of scope?

### DISCUSSION THREAD

**Discussion by do-me** [qdrant/qdrant#1472] [Ideas]
It might be out of scope for Qdrant as a *vector search* database but I can imagine some use cases where it would be nice to perform some basic math operations on the payload. 

E.g. consider your [startup search demo](https://demo.qdrant.tech/#/) where it would be nice to get e.g. the total amount of venture capital for the top 5 most related startups in your sector. 

Is this out of scope for Qdrant or would you support such a feature?

**Comment by generall** [comment]
> the total amount of venture capital for the top 5 most related startups in your sector.

I guess it would be pretty easy to do on a post-processing step, if you have such data stored somewhere.

**Reply by do-me** [reply]
Exactly, that's what I'm doing right now: simply returning the IDs and querying another DB which is ok for small instances but quickly becomes the bottleneck when data/lists becomes larger.

My idea or rather my wish would have been to avoid using another DB by having something like simplified SQL available in Qdrant for simple operations (on large data). But I assume that's out of scope for now.

**Reply by generall** [reply]
Why can't you store aggregated data in the qdrant payload?
